contracts/interfaces/IERC3156FlashLender.sol

```diff
@@ -15,25 +15,25 @@ interface IERC3156FlashLender {
   /**
     * @dev The fee to be charged for a given loan.
     * @param token The loan currency.
-    * @param amount The amount of tokens lent.
+    * @param value The amount of tokens lent.
     * @return The amount of `token` to be charged for the loan, on top of the returned principal.
     */
   function flashFee(
       address token,
-      uint256 amount
+      uint256 value
```

Comment:
Rationale on the rename:?

Reply:
just keeping the var name consistent throughout the contract